src/test/java/pageObjects/DevicePageTests.java

```diff
@@ -13,19 +13,62 @@ public void successfullyCreateDevice() {
         Assert.assertTrue(driver.findElement(By.id("device-table-row-0")).isDisplayed());
     }
 
+    @Test
+    public void successfullyEditADeviceName() {
+        devicePage.successfullyEditADeviceName();
+        String successEdit = driver.findElement(By.id("success_toaster")).getText();
+        Assert.assertEquals(successEdit, "Item updated successfully!");
+    }
+    @Test
+    public void successfullyEditADeviceSerialNo(){
+        devicePage.successfullyEditADeviceSerialNo();
+        String successEdit = driver.findElement(By.id("success_toaster")).getText();
+        Assert.assertEquals(successEdit, "Item updated successfully!");
+    }
+    @Test
+    public void successfullyEditADeviceInventoryNo(){
+        devicePage.successfullyEditADeviceInventoryNo();
+        String successEdit = driver.findElement(By.id("success_toaster")).getText();
+        Assert.assertEquals(successEdit, "Item updated successfully!");
+    }
+
+    @Test
+    public void successfullyEditADeviceInvoiceNo(){
+        devicePage.successfullyEditADeviceInvoiceNo();
+        String successEdit = driver.findElement(By.id("success_toaster")).getText();
+        Assert.assertEquals(successEdit, "Item updated successfully!");
+    }
+    @Test
+    public void successfullyEditDescription(){
+        devicePage.successfullyEditDescription();
+        String successEdit = driver.findElement(By.id("success_toaster")).getText();
+        Assert.assertEquals(successEdit, "Item updated successfully!");
+    }
```

Comment:
are we sure the item is successfully edited just by having a success message displayed?